include/c_types/path_rtdd.h

```diff
@@ -0,0 +1,51 @@
+/*PGR-GNU*****************************************************************
+File: path_rtdd.h
+
+Copyright (c) 2015 Celia Virginia Vergara Castillo
+Mail: vicky AT erosion.dev
+
+Copyright (c) 2023 Aryan Gupta
+Mail: guptaaryan1010 AT gmail.com
+
+------
+
+This program is free software; you can redistribute it and/or modify
+it under the terms of the GNU General Public License as published by
+the Free Software Foundation; either version 2 of the License, or
+(at your option) any later version.
+
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with this program; if not, write to the Free Software
+Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
+
+ ********************************************************************PGR-GNU*/
+/*! @file */
+
+#ifndef INCLUDE_C_TYPES_path_rtdd_H_
```

Comment:
Header Include guards should be all-caps. I guess that clang would report a warning in this case.